vsts/scripts/tagRunTimeImagesForRelease.sh

```diff
@@ -29,17 +29,19 @@ while read sourceImage; do
     releaseTagName=$(echo $tag | sed "s/$replaceText//g")
 
     IFS='-'
+    read -ra tagParts <<< "$tag"
```

Comment:
Just wondering if this file's logic still holds correct with the following format that you are newly creating: `oryxdevmcr.azurecr.io/public/oryx/node:10.10-Oryx-CI.20190924.3-patch1`

Reply:
This logic still holds; from the logs of a build that got to this stage:

```
Pulling the source image oryxdevmcr.azurecr.io/public/oryx/node:10.14-Oryx-CI.release-tests-patch1 ...

...

Tagging the source image with tag oryxmcr.azurecr.io/public/oryx/node:10.14-release-tests-patch1...
```